overlord/configstate/configcore/picfg.go

```diff
@@ -105,6 +106,20 @@ func (e *piConfigNotSupportedError) Error() string {
 	return fmt.Sprintf("configuring not supported: %s", e.reason)
 }
 
+var ignorePrefix = []byte("# SNAPD IGNORE\n")
+
+func piConfigFileIgnoreMarkerSet(configFile string) bool {
+	f, err := os.Open(configFile)
+	if err != nil {
+		return false
+	}
+	defer f.Close()
+
+	buf := make([]byte, len(ignorePrefix))
```

Comment:
I think we want to read the full first line and then apply a regexp to it so that we can ignore different spacing or case:

`# Snapd-Edit: no`